src/index.ts

```diff
@@ -21,16 +21,44 @@ http("scraping-rakuten-product-detail", async (req, res) => {
   const browser = await puppeteer.launch(options)
   const page = await browser.newPage()
   await page.goto(body.siteUrl)
-  await page.waitForSelector(".sale_desc")
 
-  const imageUrls = await page.$$eval("span.sale_desc img", (list) =>
-    list.map((el) => (el as HTMLImageElement).src)
-  )
+  const getImageUrls = async () => {
+    try {
+      await page.waitForSelector(".sale_desc")
+      return await page.$$eval("span.sale_desc img", (list) =>
+        list.map((el) => (el as HTMLImageElement).src)
+      )
+    } catch (e) {
+      console.log(e)
+      return []
+    }
+  }
+  const imageUrls = await getImageUrls()
+
+  const getItemDesc = async (): Promise<{
+    itemDescText?: string
+    imageUrls: string[]
+  }> => {
+    try {
+      await page.waitForSelector(".item_desc")
+
+      const itemDesc = await page.$(".item_desc")
+      const itemDescText: string | undefined = await (
+        await itemDesc?.getProperty("innerText")
+      )?.jsonValue()
```

Comment:
```suggestion
      const itemDescText: string | null = await page.$eval(
        ".item_desc",
        (el) => el.textContent
```
処理的には問題ないですが、50行目みたいに非同期処理を一回だけでもいいかなと思いました！